components/NoteAction/NoteActionComment.tsx

```diff
@@ -4,13 +4,24 @@ import requireAuth from "../../utils/hoc/requireAuth";
 import NoteAction from "./NoteAction";
 import { useEvent } from "../../ndk/NDKEventProvider";
 import { useEventReplies } from "../../ndk/hooks/useEventReplies";
+import { openSheet } from "../../store/Sheets";
+import { useDispatch } from "react-redux";
 
-const NoteActionComment = ({ onClick }) => {
+const NoteActionComment = () => {
   const { event } = useEvent();
   const replies = useEventReplies(event);
+  const dispatch = useDispatch();
+  const handleClickComment = () => {
+    dispatch(
+      openSheet({
+        sheetKey: "postSheet",
+        replyingTo: event.rawEvent(),
```

Comment:
needed to convert this to a raw event since is recommended for redux value to serializable.